composer.json

```diff
@@ -23,7 +23,7 @@
         "aws/aws-sdk-php": "3.258",
         "keboola/api-error-control": "^3.10",
         "keboola/configuration-variables-resolver": "^4.0",
-        "keboola/dockerbundle": "dev-odin-library-3",
```

Comment:
snad projde :)

Reply:
neprošlo...

Reply:
Jo ale neproslo to jen diky tomu ze se v internal api zavedl ted povinny branchType a ty testy si zakladaji joby. Jinak by to proslo bejt to vcera:) 

Upravim to

Reply:
Tak to nezapomeň zase vrátit na dev-odin-library-3

Reply:
odina jsem vrátil do hry